app/models/song.rb

```diff
@@ -4,4 +4,5 @@ class Song < ActiveRecord::Base
   validates :artist_id, presence: true
 
   belongs_to :artist
```

Comment:
So, here's a small code style point that maybe I glanced over a bit this morning:

https://github.com/IntrepidPursuits/apprentice-lectures-summer-2015/blob/master/rails/code_style/rails-code-style.md#activerecord

We like to organize our modes with associations first, then validations. So here, this would be

```
belongs_to :artist

validates :artist_id, presence: true

counter_culture :artist, touch: :true
```